lib/Heartbeat/Heartbeat.cpp

```diff
@@ -35,7 +35,7 @@ Heartbeat::Heartbeat(LED& led, int interval) : _led(led) {
   off();
 }
 
-Heartbeat::Heartbeat(int pin, int interval)
+Heartbeat::Heartbeat(uint8_t pin, unsigned int interval)
```

Comment:
This is not the file mentioned in the commit massage.

Reply:
Probably an issue introduced by one of my rebases.

Reply:
fixed.